property.go

```diff
@@ -20,6 +20,33 @@ type PropertyDefinitionConnection struct {
 	TotalCount int `graphql:"-"`
 }
 
+type PropertyInput struct {
+	Owner         IdentifierInput `json:"owner"`
+	Definition    IdentifierInput `json:"definition"`
+	Value         JSONString      `json:"value"`
+	RunValidation *bool           `json:"runValidation,omitempty"`
+}
+
+// Treating Definition (PropertyDefinition) as an IdentifierInput
+// Treating Owner (... on Service) as an IdentifierInput
+type Property struct {
+	Definition       IdentifierInput  `graphql:"definition"`
+	Owner            IdentifierInput  `graphql:"owner"`
+	ValidationErrors []OpsLevelErrors `graphql:"validationErrors"`
+	Value            JSONString       `graphql:"value"`
+}
+
+type ServicePropertiesEdge struct {
+	Cursor string    `graphql:"cursor"`
+	Node   *Property `graphql:"node"`
+}
+
+type ServicePropertiesConnection struct {
+	Edges      []ServicePropertiesEdge
```

Comment:
we don't generally uses edges - we use nodes.  Additionally the connection type is usually defined inline and not as a separate struct.

Reply:
[I used the same design as dependencies.go](https://github.com/OpsLevel/opslevel-go/blob/aced0c74dc751af84965c53e220f611a14e1f539/dependencies.go#L26-L29)

So I think that the connection type is required to be defined a struct in this file [so that it can be added to the `Service` object.](https://github.com/OpsLevel/opslevel-go/blob/aced0c74dc751af84965c53e220f611a14e1f539/service.go#L36)

I will try to refactor this to use Nodes instead of edges.

Reply:
@rocktavious I have refactored this to use `Nodes []Property` instead of edges.

Reply:
ahh the reason the use edges is because they put extra data on the edges - which i hate - so we should only do that if they did this.  Otherwise i prefer to use nodes generally.